overrides/config/ftbquests/quests/chapters/create.snbt

```diff
@@ -1,30 +1,3261 @@
+			description: [
+				"The &eMechanical Press&r is used in a variety of ways."
+				""
+				"Pressing some ingots on a &eDepot&r will turn the items into their sheet form."
+				""
+				"Compacting allows the automation of recipes where you compact an item (ie, Iron Blocks) by pressing the items in a &eBasin&r."
+			]
+			icon: "create:mechanical_press"
+			id: "40A64D59F1186585"
+			rewards: [{
+				amount: 50L
+				currency: "eternalcurrencies:coins"
+				id: "0EC401B1BD837E08"
+				ignore_reward_blocking: true
+				type: "eternalcurrencies:currency"
+			}]
+			subtitle: "Craft a &eMechanical Press"
+			tasks: [
+				{
+					id: "7EA8C9C4A51E53DC"
+					item: "create:mechanical_press"
+					type: "item"
+				}
+				{
+					id: "339D4FE6A5B2BBF7"
+					item: "create:depot"
+					type: "item"
+				}
+				{
+					id: "069AACB28B2411A5"
+					item: "create:basin"
+					type: "item"
+				}
+			]
+			title: "Mechanical Press"
+			x: 1.0d
+			y: -2.0d
+		}
+		{
+			icon: "create:wrench"
+			id: "0B00CE679871EE5D"
+			rewards: [
+				{
+					amount: 80L
+					currency: "eternalcurrencies:coins"
+					id: "08DEA4EE197C2A3E"
+					ignore_reward_blocking: true
+					type: "eternalcurrencies:currency"
+				}
+				{
+					id: "43DD117B6AF2D1C4"
+					item: {
+						Count: 1
+						id: "ftbquests:lootcrate"
+						tag: {
+							Enchantments: [{
+								id: "ars_nouveau:reactive"
+								lvl: 1
+							}]
+							HideFlags: 33
+							display: {
+								Lore: [
+									"{\"text\":\"§7§o§e§oContains a random special item!\"}"
+									"{\"text\":\"§b§7§o§bRight-Click to Open\"}"
+									"{\"text\":\"§5§6✎ §nLore Text\"}"
+									"{\"text\":\"§7§6§o§7§o\\\"An assortment of miscellaneous items of varying usefulness found within The Quest Master's storage locker and repackaged as your reward!\\\"\"}"
+									"{\"text\":\"\"}"
+								]
+								Name: "{\"text\":\"§eQuest Loot\"}"
+							}
+							"quark:RuneAttached": 1b
+							"quark:RuneColor": {
+								Count: 1b
+								id: "quark:rainbow_rune"
+							}
+							rpgloot.randomize: 0b
+							type: "quest_loot"
+						}
+					}
+					type: "item"
+				}
+			]
+			size: 0.8d
+			subtitle: "Tools Every Engineer Needs"
+			tasks: [
+				{
+					id: "0C73E35A8DBFCAF0"
+					item: "create:goggles"
+					type: "item"
+				}
+				{
+					id: "0DEB6FB45D6D13D4"
+					item: "create:wrench"
+					type: "item"
+				}
+				{
+					id: "23432FA831AF1533"
+					item: "create:brown_toolbox"
+					type: "item"
+				}
+				{
+					id: "3BD1FAAEE7077835"
+					item: {
+						Count: 1
+						id: "create:super_glue"
+						tag: {
+							Damage: 0
+						}
+					}
+					type: "item"
+				}
+			]
+			title: "&fTools of the Trade&r &5[Important]"
+			x: -3.0d
+			y: 1.0d
+		}
+		{
+			description: [
+				"The &eMechanical Saw&f can be used to cut down an entire tree in a single chop."
+				""
+				"Passing logs through a &eMechanical Saw&r will strip them and yield more planks."

```

Comment:
This area could probably note the Saw changing function based on its orientation
```suggestion
				"When facing upwards, passing Logs through a &eMechanical Saw&r will cut Items, for Logs, first Stripping them, and then cutting them into more Planks than usual!"
```